db/db.js

```diff
@@ -28,7 +28,8 @@ module.exports = function () {
         const mongoPORT = process.env.MONGO_PORT|| '27017';
         const dev_db_url = `mongodb://${mongoURL}:${mongoPORT}/climateTree`;
         //const connStr='mongodb://stories-mongodb:KKYmGrO8a2NUL6jGaawC3BPd0eM5YxRA8gVucNsCAcTikrfl0BQVaY1bqNcaslvYoHJQU9QmYqzcuw2c03hypQ==@stories-mongodb.documents.azure.com:10255/climateTree?ssl=true';
-        //const mongoDB = process.env.MONGODB_URI || dev_db_url;
+
+        const mongoDB = process.env.MONGODB_URI || dev_db_url;
```

Comment:
What doe uncommenting this do ?

Reply:
It checks whether the environment variable MONGODB_URI is set if not it'll use the dev_db_url, and while running locally it will use only on local database as the docker env variable is not set

Reply:
Lets not change anything related to db connection string now. Since the cosmosdb key has been changed. We want to find the error so lets wait and see if the database is still not wiped and then if the database is still not wipe till Saturday make this change as a new PR